codice/ddf
# DDF-4106 Migrates workspace menus to react

#### What does this PR do?
Migrates workspace menus to react.  Adds common portal component,  as well as dropdown component.

#### Who is reviewing it? 
@codice/ui 
@djblue 
@rzwiefel 
@AzGoalie 
@samuelechu 
@adimka 

#### How should this be tested?
Verify the workspace menu works as it did previously.  I've added screenshots of the components in question that should be tested to verify functional parity.

#### What are the relevant tickets?
[DDF-4106](https://codice.atlassian.net/browse/DDF-4106)

![screen shot 2018-10-30 at 8 28 27 am](https://user-images.githubusercontent.com/11984853/47729701-e5101980-dc1d-11e8-81df-dc5a94ef1343.png)
![screen shot 2018-10-30 at 8 28 20 am](https://user-images.githubusercontent.com/11984853/47729703-e5101980-dc1d-11e8-97e2-b69761569fd2.png)
![screen shot 2018-10-30 at 8 28 15 am](https://user-images.githubusercontent.com/11984853/47729704-e5101980-dc1d-11e8-9178-706cfef94b86.png)
